EventSubscriber/FetchMetadataRequestSubscriber.php

```diff
@@ -13,10 +15,15 @@ class FetchMetadataRequestSubscriber implements EventSubscriberInterface
 {
     private $fetchMetadataPolicyProvider;
     private $configProvider;
-    public function __construct(FetchMetadataPolicyProvider $fetchMetadataPolicyProvider, ConfigProviderInterface $configProvider)
+    private $logger;
+    private $requestBlockedMessage;
+
+    public function __construct(FetchMetadataPolicyProvider $fetchMetadataPolicyProvider, ConfigProviderInterface $configProvider, LoggerInterface $securityLogger)
     {
         $this->fetchMetadataPolicyProvider = $fetchMetadataPolicyProvider;
         $this->configProvider = $configProvider;
+        $this->logger = $securityLogger;
+        $this->requestBlockedMessage = '%1$s request from host %2$s blocked by Fetch Metadata Policy: %3$s';
```

Comment:
You can set this value when you declare the property, no need to wait for the constructor.
https://www.php.net/manual/en/language.oop5.properties.php